tools/wptrunner/wptrunner/executors/executorcontentshell.py

```diff
@@ -43,6 +44,9 @@ class ContentShellTestPart(ProtocolPart):
     name = "content_shell_test"
     eof_marker = "#EOF" + linesep  # Marker sent by content_shell after blocks.
 
+    if sys.platform == "win32":
+        eof_marker = (b'#EOF\n').decode()
```

Comment:
Would this be equivalent to this string literal?

```suggestion
        eof_marker = '#EOF\n'
```

Reply:
I think so, let me test this using the literal string, 
reason I used decode was to try to closely match what our old system "run_web_tests" was using.

Here's a doc with the issue + comparison with old system
https://docs.google.com/document/d/1lpVTWOtIiUrQVxY9CUb5RgJIK_O0XmjZTga89grk2X4/edit?resourcekey=0-ff5CbM-cdqKTIUIwvIj8FQ

Reply:
Code like that might have been for Python 2/3 compat at some point, but now it should be equivalent.

Reply:
ok cool, just checked results from CL and it looks good, I updated to use plain text.
https://chromium-review.googlesource.com/c/chromium/src/+/4020962